doc/getting-started/getting-started.rst

```diff
@@ -782,7 +637,7 @@ Launch the User VM
 #. Look for the line that contains the term ``virtio-blk`` and replace the
    existing image file path with your ISO image file path.  In the following
    example, the ISO image file path is
-   ``/home/acrn/acrn-work/ubuntu-18.04.6-desktop-amd64.iso``.  Here is the
+   ``/home/acrn/acrn-work/ubuntu-20.04.4-desktop-amd64.iso``.  Here is the
```

Comment:
We should use the configurator to make this change to the launch script (see previous comment).  Then we can delete all these script editing instructions completely (lines 630-676).